utils/ensureEthereumChain.ts

```diff
@@ -1,9 +1,14 @@
 import { ETH_CHAIN_ID } from "@/constants";
 import { MetaMaskInpageProvider } from "@metamask/providers";
+import { ChainOption } from "@/types";
+import { addCENNZnetToMetaMask } from "@/utils";
 
 export default async function ensureEthereumChain(
-	extension: MetaMaskInpageProvider
+	extension: MetaMaskInpageProvider,
+	chain: ChainOption
 ): Promise<void> {
+	if (chain === "CENNZnet") return addCENNZnetToMetaMask(extension);
```

Comment:
doesn't feel like this belong here, also I have not seen anywhere that use `ensureEthereumChain(extension, "CENNZnet")`